REST Bindings/openapi.yaml

```diff
@@ -6,25 +6,21 @@ info:
     name: Apache2
 
 tags:
-  - name: 'Discovery'
-    description: |
-      Endpoints to obtain information about the endpoint, such as EPCIS and CBV versions or custom vocabularies.
-  - name: 'Capture'
+  - name: 'Top-level'
     description: |
-      Endpoints to capture EPCIS events in bulk or individually.
-  - name: 'Top-level Resources'
-    description: |
-      Endpoints to browse EPCIS events by six common top-level resources:
-
+      Endpoints to browse retrieve information about EPCIS resources:
+      - events
       - electronic product codes
       - business steps
       - business locations
       - read points
       - dispositions
-
   - name: 'Events'
     description: |
-      Endpoints that expose EPCIS events as Web resources.
+      Endpoints that expose allow you to retrieve EPCIS events as Web resources.
```

Comment:
looks like expose word is not required in description how about rephrasing it to "Endpoints helps to retrieve EPCIS events as web resources."

Reply:
OK